### DISCUSSION TITLE
How to upgrade properly?

### DISCUSSION THREAD

**Discussion by wyfang** [dani-garcia/vaultwarden#2260] [Q&A]
I am deleting the docker image and container. Then re-run:

`docker run -d --name vaultwarden -e ADMIN_TOKEN=password -v /home/bitwarden/:/data/ -p 127.0.0.1:6666:80 vaultwarden/server:latest`

I don't know if this is correct? Or does it come with an upgrade?

**Comment by jjlin** [comment]
See https://github.com/dani-garcia/vaultwarden/wiki/Updating-the-vaultwarden-image
